xla/debug_options_flags.cc

```diff
@@ -1033,6 +1035,17 @@ void MakeDebugOptionsFlags(std::vector<tsl::Flag>* flag_list,
       "XLA instantiates new Gpu graphs, it evicts graphs that were not "
       "recently executed to free space on device."));
 
+  flag_list->push_back(
```

Comment:
It's better to have a singe flag to control it all, just like we have to command buffers, something like `xla_gpu_enable_va_allocations=INPUT,OUTPUT,COMMAND_BUFFER,TEMP`

1. Input - copy inputs into VA allocated buffers
2. Output - disable output buffer reuse and simple copy from VA allocated buffer into output
3. Command buffer - all buffers inside command_buffer call gets assigned VA memory space
4. TEMP - all temporaries gets assigned into VA memory space. I'd not add that, at least initially, because we will have NCCL allocated buffers, and buffers for host offloading.

Reply:
Will change to single flag